ganga/GangaCore/Runtime/GPIFunctions.py

```diff
@@ -93,3 +95,21 @@ def convert_merger_to_postprocessor(j):
         mp.process_objects.append(stripProxy(j).merger)
         stripProxy(j).postprocessors = mp
 
+def runfile(path_to_file, init_globals=None):
+    """
+    A wrapper for the runpy.run_path() function.
+    Usage:
+    Ganga In[]: runfile('myfile.py')
+    Note: If you want to run a file which includes something like:
+
+    j = Job()
+    j.submit()
+
+    then you have to run the function with init_globals=globals() in the Ganga interpreter like this
+
+     Ganga In[]: runfile('myfile.py', init_globals=globals())
```

Comment:
As we are exporting this to the Ganga API, I would make the `init_globals=globals()` the default. I actually think it should not even be an option. If a user wants anything else, it is not really a part of the Ganga API, and they can just import runpy themselves.

Reply:
Makes sense, but I had trouble making it the default as setting init_globals=globals() in the function sets it to the globals of that file rather than the namespace of the GPI interpreter. Any ideas?

Reply:
I am not quite sure what will work here. Can you try something like

```
    from GangaCore.GPI import *
    return runpy.run_path(path_to_file, init_globals=globals())
```
where the idea is that we populate the local namespace with what has been exposed to the API of Ganga (called the GPI).

Reply:
Did you try this?